tests/regression/classification/test_classification.py

```diff
@@ -0,0 +1,266 @@
+"""Tests for Classification with OTX CLI"""
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+import os
+
+import pytest
+
+from otx.cli.registry import Registry
+from otx.cli.utils.tests import (
+    nncf_eval_testing,
+    nncf_optimize_testing,
+    otx_deploy_openvino_testing,
+    otx_eval_deployment_testing,
+    otx_regression_testing,
+    otx_train_testing,
+    pot_eval_testing,
+    pot_optimize_testing,
+)
+from tests.regression.regression_test_helpers import load_regression_configuration
+from tests.test_suite.e2e_test_system import e2e_pytest_component
+
+otx_dir = os.getcwd()
+
+templates = Registry("otx/algorithms/classification").filter(task_type="CLASSIFICATION").templates
+templates_ids = [template.model_template_id for template in templates]
+
+multi_class_regression_config = load_regression_configuration(otx_dir, "classification", "supervised", "multi_class")
+multi_class_data_args = multi_class_regression_config["data_path"]
+
+
+class TestRegressionMultiClassClassification:
+    @e2e_pytest_component
+    @pytest.mark.parametrize("template", templates, ids=templates_ids)
+    def test_otx_train(self, template, tmp_dir_path):
+        tmp_dir_path = tmp_dir_path / "multi_class_cls"
+        otx_train_testing(template, tmp_dir_path, otx_dir, multi_class_data_args)
+        otx_regression_testing(
+            template, tmp_dir_path, otx_dir, multi_class_data_args, multi_class_regression_config["model_criteria"]
+        )
+
+    @e2e_pytest_component
+    @pytest.mark.parametrize("template", templates, ids=templates_ids)
+    def test_otx_train_semisl(self, template, tmp_dir_path):
+        tmp_dir_path = tmp_dir_path / "multi_class_cls/test_semisl"
+        config_semisl = load_regression_configuration(otx_dir, "classification", "semi_supervised", "multi_class")
+        args_semisl = config_semisl["data_path"]
+
+        args_semisl["train_params"] = []
+        args_semisl["train_params"].extend(["--algo_backend.train_type", "SEMISUPERVISED"])
```

Comment:
maybe we can avoid assigning to the empty list first?

Reply:
I agree. In this time, I just want to make fast implementation. :)

Reply:
So, I might miss many things. Please give your opinion 👍